lnbits/core/services.py

```diff
@@ -248,16 +249,11 @@ class PaymentKwargs(TypedDict):
                     conn=conn,
                 )
                 wallet = await get_wallet(wallet_id, conn=conn)
-                if wallet:
-                    await websocketUpdater(
-                        wallet_id,
-                        json.dumps(
-                            {
-                                "wallet_balance": wallet.balance or None,
-                                "payment": payment._asdict(),
-                            }
-                        ),
-                    )
+                updated = await get_wallet_payment(
```

Comment:
why is `get_wallet_payment()` call needed?

Reply:
because the above `update_payment_details` arent reflected otherwise. The `new_payment` variable could be updated instead aswell